src/ctype/CTypeMetadata.spec.ts

```diff
@@ -0,0 +1,46 @@
+import CType from './CType'
+import ICType from '../types/CType'
+import CTypeMetadata from './CTypeMetadata'
+import * as CTypeUtils from './CTypeUtils'
+import { CTypeWrapperMetadata } from './CTypeSchema'
+import ICTypeMetadata from '../types/CTypeMetadata'
+
+jest.mock('../blockchainApiConnection/BlockchainApiConnection')
+
+describe('CType', () => {
+  const ctype = new CType({
+    schema: {
+      $id: 'http://example.com/ctype-1',
+      $schema: 'http://kilt-protocol.org/draft-01/ctype#',
+      properties: {
+        'first-property': { type: 'integer' },
+        'second-property': { type: 'string' },
+      },
+      type: 'object',
+    },
+  } as ICType)
+
+  const ctypeHash = ctype.hash
+
+  const ctypeMetadata = {
+    title: { default: 'string' },
+    description: { default: 'string' },
+    properties: {
+      'first-property': { type: 'integer' },
+      'second-property': { type: 'string' },
+    },
+  } as ICTypeMetadata['metadata']
```

Comment:
It is preferred to define the type like this:
```
const ctypeMetadata: ICTypeMetadata['metadata'] = { ... }
```
instead of with `as`.

The reason is, that with `as`, you are just forcing the type, whereas by defining it like I did here, Typescript actually does a type check